cmd/ndm_daemonset/probe/udevprobe.go

```diff
@@ -360,3 +364,79 @@ func (up *udevProbe) free() {
 		up.udevEnumerate.UnrefUdevEnumerate()
 	}
 }
+
+func (up *udevProbe) listenUdevEventMonitor() {
+	for {
+		event := <-up.udeveventSubscription.Events()
```

Comment:
Do we need to get an instance of the target channel on every iteration?

Reply:
fixed